src/main/resources/db/migration/v0.30.0.1__linkml_v2.0.0_update.sql

```diff
@@ -0,0 +1,2574 @@
+ALTER TABLE wbbtterm DROP CONSTRAINT wbbtterm_pkey;
+ALTER TABLE wbbtterm_aud DROP CONSTRAINT IF EXISTS wbbtterm_aud_pkey;
+ALTER TABLE wblsterm DROP CONSTRAINT wblsterm_pkey;
+ALTER TABLE wblsterm_aud DROP CONSTRAINT IF EXISTS wblsterm_aud_pkey;
+ALTER TABLE wbphenotypeterm DROP CONSTRAINT wbphenotypeterm_pkey;
+ALTER TABLE wbphenotypeterm_aud DROP CONSTRAINT wbphenotypeterm_aud_pkey;
+ALTER TABLE xbaterm DROP CONSTRAINT xbaterm_pkey;
+ALTER TABLE xbaterm_aud DROP CONSTRAINT IF EXISTS xbaterm_aud_pkey;
+ALTER TABLE xbedterm DROP CONSTRAINT xbedterm_pkey;
+ALTER TABLE xbedterm_aud DROP CONSTRAINT IF EXISTS xbedterm_aud_pkey;
+ALTER TABLE xbsterm DROP CONSTRAINT xbsterm_pkey;
+ALTER TABLE xbsterm_aud DROP CONSTRAINT IF EXISTS xbsterm_aud_pkey;
+ALTER TABLE xcoterm DROP CONSTRAINT xcoterm_pkey;
+ALTER TABLE xcoterm_aud DROP CONSTRAINT IF EXISTS xcoterm_aud_pkey;
+ALTER TABLE xpoterm DROP CONSTRAINT xpoterm_pkey;
+ALTER TABLE xpoterm_aud DROP CONSTRAINT IF EXISTS xpoterm_aud_pkey;
+ALTER TABLE xsmoterm DROP CONSTRAINT xsmoterm_pkey;
+ALTER TABLE xsmoterm_aud DROP CONSTRAINT IF EXISTS xsmoterm_aud_pkey;
+ALTER TABLE zecoterm DROP CONSTRAINT zecoterm_pkey;
+ALTER TABLE zecoterm_aud DROP CONSTRAINT IF EXISTS zecoterm_aud_pkey;
+ALTER TABLE zfaterm DROP CONSTRAINT zfaterm_pkey;
+ALTER TABLE zfaterm_aud DROP CONSTRAINT IF EXISTS zfaterm_aud_pkey;
+ALTER TABLE zfsterm DROP CONSTRAINT zfsterm_pkey;
+ALTER TABLE zfsterm_aud DROP CONSTRAINT IF EXISTS zfsterm_aud_pkey;
+
+-- Create tables
+
+CREATE TABLE auditedobject (
+	id bigint PRIMARY KEY,
+	old_id bigint,
+	old_curie varchar(255),
+	tablename varchar(255),
+	createdby_id bigint,
+	updatedby_id bigint,
+	datecreated timestamp without time zone,
+	dateupdated timestamp without time zone,
+	dbdatecreated timestamp without time zone,
+	dbdateupdated timestamp without time zone,
+	internal boolean NOT NULL DEFAULT false,
+	obsolete boolean NOT NULL DEFAULT false
+	);
+
+CREATE TABLE auditedobject_aud (
+	id bigint NOT NULL,
+	createdby_id bigint,
+	updatedby_id bigint,
+	datecreated timestamp without time zone,
+	dateupdated timestamp without time zone,
+	dbdatecreated timestamp without time zone,
+	dbdateupdated timestamp without time zone,
+	internal boolean,
+	obsolete boolean,
+	rev integer NOT NULL,
+	revtype smallint,
+	PRIMARY KEY (id, rev)
+	);	
+	
+CREATE TABLE curieobject (
+	id bigint PRIMARY KEY,
+	curie varchar(255)
+	);
+
+CREATE TABLE curieobject_aud (
+	id bigint NOT NULL,
+	curie varchar(255),
+	rev integer NOT NULL,
+	PRIMARY KEY (id, rev)
+	);
+	
+CREATE TABLE submittedobject (
+	id bigint PRIMARY KEY,
+	modentityid varchar(255),
+	modinternalid varchar(255),
+	dataprovider_id bigint
+	);
+
+CREATE TABLE submittedobject_aud (
+	id bigint NOT NULL,
+	modentityid varchar(255),
+	modinternalid varchar(255),
+	dataprovider_id bigint,
+	rev integer NOT NULL,
+	PRIMARY KEY (id, rev)
+	);
+	
+CREATE SEQUENCE auditedobject_seq START WITH 200250000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
```

Comment:
For this migration we might want to set the increment to 1 and change it back to 50 when we finish. Hibernate caches 50 ids so that it doesn't have to call the seq all the time. If we don't I think we will have huge gaps in ids here. Not sure yet... going to run this on my system now.

Reply:
Done